server/sonar-main/src/main/java/org/sonar/application/config/JdbcSettings.java

```diff
@@ -136,19 +136,15 @@ private static void warnIfUrlIsSet(int port, String existing, String expectedUrl
     if (isNotEmpty(existing)) {
       Logger logger = LoggerFactory.getLogger(JdbcSettings.class);
       if (expectedUrl.equals(existing)) {
-        logger.warn("To change H2 database port, only property '{}' should be set (which current value is '{}'). " +
-          "Remove property '{}' from configuration to remove this warning.",
-          JDBC_EMBEDDED_PORT, port,
-          JDBC_URL);
+        logger.warn("To change H2 database port, only property '" + JDBC_EMBEDDED_PORT + "' should be set (which current value is '{}'). " +
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use the built-in formatting to construct this argument. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2629)